plugins/nginx-vhosts/functions

```diff
@@ -14,6 +14,12 @@ validate_nginx() {
   set -e
   if [[ "$exit_code" -ne "0" ]]; then
     sudo /usr/sbin/nginx -t
+    shopt -s nullglob
+    local conf_file
+    for conf_file in $DOKKU_ROOT/*/nginx.conf; do
+      dokku_log_verbose "validate_nginx failed. contents of $conf_file below..."
+      cat "$conf_file"
+    done
```

Comment:
This helped debug my bogus config generation. Though it'd be nice in the wild too.